oidc-controller/api/core/config.py

```diff
@@ -0,0 +1,135 @@
+import logging
+import os
+import requests
+import json
+
+from enum import Enum
+from functools import lru_cache
+from typing import Optional
+
+from pydantic import BaseSettings, PostgresDsn
+
+logger = logging.getLogger(__name__)
+
+
+class EnvironmentEnum(str, Enum):
+    PRODUCTION = "production"
+    LOCAL = "local"
+
+
+class GlobalConfig(BaseSettings):
+    TITLE: str = "Traction"
+    DESCRIPTION: str = "A digital wallet solution for organizations"
```

Comment:
We may want to be able to override these at runtime

Reply:
fixed